app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,5 @@
+#home controller for first page 
+class StaticPagesController < ApplicationController
+  def home
+  end
```

Comment:
No need to declare this method as it doesn't containing anything, You can just make a view file